pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -218,7 +219,7 @@ func initializeDirectory(path string) error {
 
 // genCerts create ca etcd karmada cert
 func (i *CommandInitOption) genCerts() error {
-	notAfter := time.Now().Add(cert.Duration365d).UTC()
```

Comment:
There is more than one place referencing this default variable,  please check if we get it from the option.

Reply:
Thanks for pointing that, it has some problems and I am going to fix it.

Reply:
`Duration365d` is used here, and it only work when `cfg.NotAfter == nil`, but the `cfg.NotAfter` has been filled when `NewCertConfig`.

https://github.com/karmada-io/karmada/blob/b1f8b09d073d9f856aef90297e5ae3f7c780ee00/pkg/karmadactl/cmdinit/cert/cert.go#L110-L113

https://github.com/karmada-io/karmada/blob/b1f8b09d073d9f856aef90297e5ae3f7c780ee00/pkg/karmadactl/cmdinit/cert/cert.go#L234-L244

https://github.com/karmada-io/karmada/blob/b1f8b09d073d9f856aef90297e5ae3f7c780ee00/pkg/karmadactl/cmdinit/kubernetes/deploy.go#L233-L234